projects/ccf-database/src/lib/hubmap/hubmap-data.ts

```diff
@@ -228,7 +228,7 @@ export class HuBMAPTissueBlock {
       if (descendant.entity_type === 'Dataset') {
         const dataset = this.getDataset(descendant, assetsApi, portalUrl, serviceToken);
 
-        const sectionId = get(descendant, ['metadata', 'metadata', 'tissue_id']) as string;
+        const sectionId = get(descendant, ['ingest_metadata', 'ingest_metadata', 'tissue_id']) as string;
```

Comment:
You need to check the actual data on this. I suspect the second ingest_metadata is actually metadata. Look up how `get` works on lodash if you don't already know.